hardhat.config.js

```diff
@@ -16,6 +19,71 @@ task("accounts", "Prints the list of accounts", async (taskArgs, hre) => {
   }
 });
 
+const temp_file = "temp_file.txt";
+const temp_time = "temp_time.txt";
+
+task("change-time", "Change startTime", async (taskArgs, hre) => {
```

Comment:
I feel that supporting dynamic start times might be too complicated for us. A simpler approach would be to just set the start time right before deployment manually.